runtime/lua/vim/lsp/util.lua

```diff
@@ -150,8 +188,9 @@ end
 ---@private
 --- Position is a https://microsoft.github.io/language-server-protocol/specifications/specification-current/#position
 --- Returns a zero-indexed column, since set_lines() does the conversion to
+---@param encoding string utf-8|utf-16|utf-32|nil defaults to utf-16
 --- 1-indexed
-local function get_line_byte_from_position(bufnr, position, offset_encoding)
+local function get_line_byte_from_position(bufnr, position, encoding)
```

Comment:
why change from offset encoding? that's what it is in the extension to the protocol that is being adopted (offsetEncoding)

Reply:
I see, I thought I would keep it general just in case we have a need to use this in a different context. Like if there's a new protocol extension that adds a field like `length_encoding` or something?

Reply:
Hmm yeah so this was a result of a search and replace so it's unlikely that here in particular it would be anything other than an offset encoding. I could go through and make these parameter names more specific if you like.

Reply:
I just, in general, would prefer we keep renaming/refactoring to a minimum if it's not on the direct path to addressing the bug, and also try to preserve naming conventions from other parts of the codebase.

Reply:
+1, please revert unrelated changes.

Reply:
Reverted.